physionet-django/project/validators.py

```diff
@@ -5,6 +5,7 @@
 
 MAX_FILENAME_LENGTH = 100
 MAX_PROJECT_SLUG_LENGTH = 30
+MAX_FILENAME_LENGTH_GCS_BUCKET_OBJECT = 256
```

Comment:
Last tiny thing - I'd unpack this name a little.

```suggestion
MAX_GCS_OBJECT_NAME_LENGTH = 256
```

Reply:
`MAX_GCS_OBJECT_NAME_LENGTH` is a way better name. i like it. Its consistent with the others and easy on the eye and not too long :D